opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -130,14 +130,24 @@ message ProfilesDictionary {
   // string_table[0] must always be "" and present.
   repeated string string_table = 5;
 
-  // A common table for attributes referenced by various messages.
-  // It is a collection of key/value pairs. Note, global attributes
-  // like server name can be set using the resource API. Examples of attributes:
+  // A common table for attributes referenced by the Profile, Sample, Mapping
+  // and Location messages below through attribute_indices field. Each entry is
+  // a key/value pair with an optional unit. Since this is a dictionary table,
+  // multiple entries with the same key may be present, unlike direct attribute
+  // tables like Resource.attributes. The referencing attribute_indices fields,
+  // though, do maintain the key uniqueness requirement.
   //
+  // It's recommended to use attributes for variables with bounded set of
+  // values, such as categorical variables
+  // (https://en.wikipedia.org/wiki/Categorical_variable). Using an attribute
```

Comment:
Nit: Otel typically uses the term "cardinality" when referring to the number of unique values (and you also use it in the PR title), so I'd use that more commonly used term. That is, unless there is some nuance that I am missing.

I haven't heard of "Categorical variables" before (and thanks for the Wikipedia link). Can be just me :-)

Reply:
Ah, let me add the cardinality term back

Reply:
Thanks!